packages/expo-cli/src/credentials/local/credentialsJson.ts

```diff
@@ -64,7 +64,7 @@ async function readAndroidAsync(projectDir: string): Promise<AndroidCredentials>
   const keystoreInfo = credentialsJson.android.keystore;
   return {
     keystore: {
-      keystore: await fs.readFile(keystoreInfo.keystorePath, 'base64'),
+      keystore: await fs.readFile(path.join(projectDir, keystoreInfo.keystorePath), 'base64'),
```

Comment:
this change will not work with absolute paths, are you sure you want to do this?